browser/brave_ads/tooltips/ads_captcha_tooltip.h

```diff
@@ -0,0 +1,46 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_BRAVE_ADS_TOOLTIPS_ADS_CAPTCHA_TOOLTIP_H_
+#define BRAVE_BROWSER_BRAVE_ADS_TOOLTIPS_ADS_CAPTCHA_TOOLTIP_H_
+
+#include <memory>
+#include <string>
+
+#include "brave/browser/ui/brave_tooltips/brave_tooltip.h"
+
+class Profile;
+
+namespace brave_ads {
+
+extern const char kScheduledCaptchaTooltipId[];
+
+class AdsCaptchaTooltip : public brave_tooltips::BraveTooltip {
+ public:
+  AdsCaptchaTooltip(Profile* profile,
+                    const brave_tooltips::BraveTooltipAttributes& attributes,
+                    const std::string& payment_id,
+                    const std::string& captcha_id);
+  ~AdsCaptchaTooltip() override;
+
+  AdsCaptchaTooltip(const AdsCaptchaTooltip&) = delete;
+  AdsCaptchaTooltip& operator=(const AdsCaptchaTooltip&) = delete;
+
+  std::string payment_id() const { return payment_id_; }
```

Comment:
How about returning reference-to-`const`s for `payment_id()` and `captcha_id()`?

Reply:
Great idea, fixed.